src/azure-cli/azure/cli/command_modules/acr/_docker_utils.py

```diff
@@ -217,18 +217,21 @@ def _get_credentials(cmd,  # pylint: disable=too-many-statements
     url = 'https://' + login_server + '/v2/'
     try:
         challenge = requests.get(url, verify=(not should_disable_connection_verify()))
-        if challenge.status_code in [403]:
-            raise CLIError("Looks like you don't have access to registry '{}'. ".format(login_server) +
-                           "To see configured firewall rules, run" +
-                           " 'az acr show --query networkRuleSet --name {}'. ".format(registry_name) +
-                           "Details: https://docs.microsoft.com/azure/container-registry/container-registry-health-error-reference#connectivity_forbidden_error")  # pylint: disable=line-too-long
+        if challenge.status_code == 403:
+            raise CLIError("Looks like you don't have access to registry '{}'. "
+                           "To see configured firewall rules, run 'az acr show --query networkRuleSet --name {}'. "
+                           "Please refer to https://aka.ms/acr/errors#connectivity_forbidden_error for more information."  # pylint: disable=line-too-long
```

Comment:
Pylint (`azdev style acr`) was complaining about line too long. 
So, I had to add `# pylint: disable=line-too-long`